.config/samples-config-v1.json

```
@@ -0,0 +1,175 @@
+{
+    "filterOptions": {
+        "capabilities": [
+            "Word",
+            "Excel",
+            "PowerPoint"
+        ],
+        "languages": [
+            "TS",
+            "JS"
+        ],
+        "technologies": [
+            "Custom Function",
+            "SSO",
+            "Graph"
+        ]
+    },
+    "samples": [
+        {
+            "id": "Excel-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Excel Hello World Task Pane Add-in",
+            "shortId": "Excel Hello World",
+            "shortDescription": "A simple Excel Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Excel Task Pane Add-in using JavaScript.",
+            "types": [
+                "Excel"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Excel"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Word Hello World Task Pane Add-in",
+            "shortId": "Word Hello World",
+            "shortDescription": "A simple Word Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Word Task Pane Add-in using JavaScript.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-AIGC",
+            "onboardDate": "2024-03-15",
+            "title": "Create a Word AIGC Add-in",
+            "shortId": "Word AIGC",
+            "shortDescription": "Inserting content generated by AI or predefined into document.",
+            "fullDescription": "This Add-in shows capabilities to insert content generated by AI or predefined into document and customize the style of document.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "AIGC",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-WritingAssistant",
+            "onboardDate": "2024-03-06",
+            "title": "Word Writing Assistant",
+            "shortId": "Word Writing Assistant",
+            "shortDescription": "A Word Add-in to assist writing with word document.",
```

Some branding notes:

- It's "Word add-in", not "Word Add-in". The brand term is "Office Add-ins", so in this case we're talking about an add-in for Word.
- Always capitalize "Word" when referring to the application.
```suggestion
            "shortDescription": "A Word add-in to assist with writing a Word document.",
```